lib/plugins/audit.js

```diff
@@ -25,6 +25,10 @@ var HttpError = require('restify-errors').HttpError;
 function auditLogger(options) {
     assert.object(options, 'options');
     assert.object(options.log, 'options.log');
+    //use server object to emit log data
+    var server = options.server;
+    //use logMetrics object to store certain period of log records
+    var logMetrics = options.logMetrics;
```

Comment:
The LogMetric class is overkill. Just have the user pass in a ringbuffer object themselves, and directly use that instead.